ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/Rao.java

```diff
@@ -0,0 +1,197 @@
+            = Suppliers.memoize(() -> new ServiceLoaderCache<>(RaoProvider.class).getServices());
+
+    /**
+     * A RA optimisation runner is responsible for providing convenient methods on top of {@link RaoProvider}:
+     * several variants of synchronous and asynchronous run with default parameters.
+     */
+    public static class Runner implements Versionable {
+
+        private final RaoProvider provider;
+
+        public Runner(RaoProvider provider) {
+            this.provider = Objects.requireNonNull(provider);
+        }
+
+        public CompletableFuture<RaoComputationResult> runAsync(Network network, Crac crac, String workingStateId, ComputationManager computationManager, RaoParameters parameters) {
+            Objects.requireNonNull(workingStateId);
+            Objects.requireNonNull(parameters);
+            return provider.run(network, crac, workingStateId, computationManager, parameters);
+        }
+
+        public CompletableFuture<RaoComputationResult> runAsync(Network network, Crac crac, ComputationManager computationManager, RaoParameters parameters) {
+            return runAsync(network, crac, network.getVariantManager().getWorkingVariantId(), computationManager, parameters);
+        }
+
+        public CompletableFuture<RaoComputationResult> runAsync(Network network, Crac crac, RaoParameters parameters) {
+            return runAsync(network, crac, LocalComputationManager.getDefault(), parameters);
+        }
+
+        public CompletableFuture<RaoComputationResult> runAsync(Network network, Crac crac) {
+            return runAsync(network, crac, RaoParameters.load());
+        }
+
+        public RaoComputationResult run(Network network, Crac crac, String workingStateId, ComputationManager computationManager, RaoParameters parameters) {
+            Objects.requireNonNull(workingStateId);
+            Objects.requireNonNull(parameters);
+            return provider.run(network, crac, workingStateId, computationManager, parameters).join();
+        }
+
+        public RaoComputationResult run(Network network, Crac crac, ComputationManager computationManager, RaoParameters parameters) {
+            return run(network, crac, network.getVariantManager().getWorkingVariantId(), computationManager, parameters);
+        }
+
+        public RaoComputationResult run(Network network, Crac crac, RaoParameters parameters) {
+            return run(network, crac, LocalComputationManager.getDefault(), parameters);
+        }
+
+        public RaoComputationResult run(Network network, Crac cracFile) {
+            return run(network, cracFile, RaoParameters.load());
+        }
+
+        @Override
+        public String getName() {
+            return provider.getName();
+        }
+
+        @Override
+        public String getVersion() {
+            return provider.getVersion();
+        }
+    }
+
+    /**
+     * Get a runner for optimising RA named {@code name}. In the case of a null {@code name}, default
```

Comment:
This description is not clear . I think it should be "Get a runner for RAO named ..."

Reply:
done